discovery/db_discovery.go

```diff
@@ -45,7 +48,7 @@ func NewDBOrchestratorPoolCache(node *core.LivepeerNode) *DBOrchestratorPoolCach
 }
 
 func (dbo *DBOrchestratorPoolCache) GetURLs() []*url.URL {
-	orchs, err := dbo.node.Database.SelectOrchs()
+	orchs, err := dbo.node.Database.SelectOrchs(nil)
```

Comment:
This should have a filter as well (missed this case during our earlier discussions)

Reply:
Fixed in https://github.com/livepeer/go-livepeer/pull/939/commits/34be3ca41fbaed6a94a78b807eada7fd2508e370